hypervisor/debug/uart16550.c

```diff
@@ -14,20 +14,25 @@
 
 #define MAX_BDF_LEN 8
 
+union uart_addr {
+	uint16_t pio_base_paddr;
+	uint32_t mmio_base_vaddr;
+};
+
 #if defined(CONFIG_SERIAL_PIO_BASE)
 static bool serial_port_mapped = true;
 static bool uart_enabled = true;
-static uint64_t uart_base_address = CONFIG_SERIAL_PIO_BASE;
+static uint16_t uart_pio_base_paddr = CONFIG_SERIAL_PIO_BASE;
 static char pci_bdf_info[MAX_BDF_LEN + 1U];
 #elif defined(CONFIG_SERIAL_PCI_BDF)
 static bool serial_port_mapped;
 static bool uart_enabled = true;
-static uint64_t uart_base_address;
+static uint16_t uart_pio_base_paddr;
```

Comment:
union uart_addr  should be used here to replace the old variable. We don't need to add a new one. Right?

Reply:
Yes. Will remove it.